Sources/MapboxCoreNavigation/CoreConstants.swift

```diff
@@ -40,6 +40,13 @@ public var RouteControllerMinimumDurationRemainingForProactiveRerouting: TimeInt
  */
 public var RouteControllerProactiveReroutingInterval: TimeInterval = 120
 
+/**
+ The distance threshold to the current leg destination to anounce the arrival of the waypoint.
+ 
+ During active navigation, when the remaining distance of current route leg is equal or smaller than the `RouteControllerDistanceForArrival`, the arrival event at current leg waypoint would be announced.
```

Comment:
It is not clear that the arrival event still depends on the `.complete` status from NavNative (that depends on the route response from Directions).

Reply:
I see, that's true. Should we mention that it's not guaranteed that arrival will be triggered at this point as this variable can only decrease the remaining distance?

Reply:
Thanks for the suggestion, I'll update the documentation.